server/app.js

```diff
@@ -12,6 +12,12 @@ import pingRouter from './routes/ping';
 
 const app = express();
 
+const LoggerMiddleware = (req, res, next) => {
+  console.log(`Logged  ${req.url}  ${req.method} -- ${new Date()}`);
+  next();
+};
+app.use(LoggerMiddleware);
+
```

Comment:
is this for listing the routes?

Reply:
This just logs the requested route and method. I found it helpful but no need to keep it. As for a rake-routes-like terminal command, looks like there are a few random packages out there but I haven't looked into them yet. Nothing built-in.